README.md

```diff
@@ -95,6 +95,28 @@ table.add_rows(
 )
 ```
 
+#### Some rows at once (with divider)
+
+When you have a list of rows, you can add them in one go with `add_rows`:
+
+```python
+table.field_names = ["City name", "Area", "Population", "Annual Rainfall"]
+table.add_rows(
+    [
+        ["Adelaide", 1295, 1158259, 600.5],
+        ["Brisbane", 5905, 1857594, 1146.4],
+        ["Darwin", 112, 120900, 1714.7]
+    ], divider=True
```

Comment:
```suggestion
    ],
    divider=True,
)
```

And can we move this down to the "Adding sections to a table" part? Maybe use example code that produces the same output as the existing one.

Reply:
Good point, I'll mix it into that existing example.